crates/sui-json-rpc/src/indexer_api.rs

```diff
@@ -123,6 +122,16 @@ impl<R: ReadApiServer> IndexerApi<R> {
             Err(_) => bail!("Resources exhausted"),
         }
     }
+
+    async fn get_latest_checkpoint_timestamp_ms(&self) -> StateReadResult<u64> {
```

Comment:
@amnn does this seem like the right way to get the latest checkpoint's timestamp?
I am a bit confused between `read_api` and `state`, but it looks like `read_api` is using `state` behind the scenes? 
If you have some extra context (or docs), would be great for me to understand what's going on.